### DISCUSSION TITLE
Support on custom vector store

### DISCUSSION THREAD

**Discussion by shintiger** [open-webui/open-webui#3834] [General]
**Is your feature request related to a problem? Please describe.**
I have existing vector store (qdrant) stored a set of parsed content, need to share with Open WebUI

**Describe the solution you'd like**
Support on custom vector store like setting external ollama address, I am using qDrant

**Describe alternatives you've considered**
ENV setting is also fine

**Additional context**

**Comment by ErroneousBosch** [comment]
This would be very handy, as a vector store could be updated externally using using a different workflow that runs in something like a dedicated docker script or n8n, and used by multiple clients.
